### DISCUSSION TITLE
is there a way to opt out of one rule only?

### DISCUSSION THREAD

**Discussion by Sboonny** [mattpocock/ts-reset#107] [Q&A] [answered]
looking at [rules](https://github.com/total-typescript/ts-reset#rules), all seems good and works perfectly in the application, but we have a problem `array-include`.

```ts
import "@total-typescript/ts-reset/array-includes"
```

Reading the rules, the way around this is to import every other rule except `array-includes`, but I don't want that because what if there is a new rule added, and I want it to be applied after updating the package, there will be a chance that I will miss that new rule.

so is there a way to opt out of a rule?

**Comment by mattpocock** [comment, marked as answer]
Unfortunately not!
